_source/atoms/icon/IconContainer.js

```diff
@@ -0,0 +1,12 @@
+import { connect } from 'react-redux';
+import Component from './Icon';
+
+export const mapStateToProps = (state) => ({
+  darkMode: state.user.darkMode
+});
```

Comment:
Just now that I see this again, the dark mode basically just changes colors as all the other color-picker does, doesn't it? Why do we need to touch basic components to make it happen? What is the difference between dark mode and color changes?